requirements/doc.txt

```diff
@@ -4,9 +4,8 @@
 #
 #    pip-compile --upgrade -o requirements/doc.txt requirements/doc.in
 #
-alabaster==0.7.12         # via sphinx
```

Comment:
All the Sphinx requirements got dropped; this happened in the edx-platform job also, but not when running make upgrade locally.  We need to fix this before merging.